test/e2e/istio_test.go

```diff
@@ -0,0 +1,212 @@
+// +build istio
+
+package e2e
+
+import (
+	"context"
```

Comment:
I can never remember the correct import order off the top of my head, but this does not look correct to me.  @jpkrohling  can you remind me what this is supposed to be?

Reply:
Only 3 blocks, being first the stdlib, then external, then internal :-)

Reply:
Done